enginetest/testdata.go

```diff
@@ -78,6 +78,58 @@ func createSubsetTestData(t *testing.T, harness Harness, includedTables []string
 	var table sql.Table
 	var err error
 
+	if includeTable(includedTables, "point_table") {
+		wrapInTransaction(t, myDb, harness, func() {
+			table, err = harness.NewTable(myDb, "point_table", sql.NewPrimaryKeySchema(sql.Schema{
+				{Name: "i", Type: sql.Int64, Source: "point_table", PrimaryKey: true},
+				{Name: "p", Type: sql.Point, Source: "point_table"},
+			}))
+
+			if err == nil {
+				InsertRows(t, NewContext(harness), mustInsertableTable(t, table),
+					sql.NewRow(5, "point(1,2)"),
```

Comment:
I would also expect this to be
```suggestion
					sql.NewRow(5, sql.PointValue{X: 1, Y: 2}),
```